tools/spec-gen-sdk/src/automation/entrypoint.ts

```diff
@@ -68,6 +69,7 @@ export const getSdkAutoContext = async (options: SdkAutoOptions): Promise<SdkAut
 
   const fullLogFileName = path.join(options.workingFolder, 'full.log');
   const filteredLogFileName = path.join(options.workingFolder, 'filtered.log');
+  const logHtml = path.join(options.workingFolder, 'logIndex.html');
```

Comment:
Can you rename to `{spec-gen-sdk-language}-result.html`, such as `spec-gen-sdk-java-result.html`?